lib/ns.c

```diff
@@ -35,17 +35,23 @@
 
 #include "command.h"
 #include "vty.h"
+#include "vrf.h"
 
 DEFINE_MTYPE_STATIC(LIB, NS, "Logical-Router")
 DEFINE_MTYPE_STATIC(LIB, NS_NAME, "Logical-Router Name")
 
 static __inline int ns_compare(const struct ns *, const struct ns *);
 static struct ns *ns_lookup(ns_id_t);
+static struct ns *ns_lookup_name(const char *);
 
 RB_GENERATE(ns_head, ns, entry, ns_compare)
 
 struct ns_head ns_tree = RB_INITIALIZER(&ns_tree);
 
+static struct ns *default_ns;
+static int ns_current_ns_fd;
+static int ns_default_ns_fd;
+
```

Comment:
I believe the fd should be initialized to -1

Reply:
same.